index.html

```diff
@@ -1,158 +1,143 @@
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300;600;800&display=swap" rel="stylesheet" />
+  <!-- Fontawesome Library -->
+  <link rel="stylesheet" href="https://site-assets.fontawesome.com/releases/v6.0.0/css/all.css" />
+  <!-- Shared CSS Styles File -->
+  <link rel="stylesheet" href="./css/common.css" />
+  <!-- Page Custom CSS File -->
+  <link rel="stylesheet" href="./css/main.css" />
+  <title>Countries Info</title>
+</head>
+
+<body>
+  <!-- Scroll Top -->
+  <button type="button" aria-label="Scroll Top Button" class="button scroll-top flex flex-ai-c flex-jc-c scale-effect">
+    <i class="fa-regular fa-chevron-up icon"></i>
+  </button>
+  <!-- Header -->
+  <header class="header">
+    <div class="container flex flex-jc-sb flex-ai-c">
+      <div class="logo">
+        <a href="index.html">
+          <h1>Where in the world?</h1>
+        </a>
       </div>
-    </header>
-    <!-- Filters -->
+      <button type="button" aria-label="Theme Switcher Button" class="theme-toggle flex flex-jc-sb flex-ai-c" onclick="changeColorOfPage()">
+        <i class="fa-regular fa-moon theme-icon"></i>
+        <span class="theme-text">Dark Mode</span>
+      </button>
+    </div>
+  </header>
+  <!-- Filters -->
 
-    <section class="filters">
-      <div class="container">
-        <div class="search-wrapper">
-          <i class="fa-regular fa-magnifying-glass search-icon"></i>
-          <input
-            type="text"
-            class="search-input"
-            placeholder="Search for a country..."
-          />
+  <section class="filters">
+    <div class="container">
+      <div class="search-wrapper">
+        <i class="fa-regular fa-magnifying-glass search-icon"></i>
+        <input type="text" class="search-input" placeholder="Search for a country..." oninput="filterBySerach(event)"/>
+      </div>
+      <div class="dropdown-wrapper" onclick="openDropDown()">
+        <div class="dropdown-header flex flex-jc-sb flex-ai-c">
+          <span>Filter by Region</span>
+          <i class="fa-regular fa-chevron-down icon"></i>
         </div>
-        <div class="dropdown-wrapper">
-          <div class="dropdown-header flex flex-jc-sb flex-ai-c">
-            <span>Filter by Region</span>
-            <i class="fa-regular fa-chevron-down icon"></i>
+        <div class="dropdown-body">
+          <ul>
+            <li data-region="all" onclick="filterByRegion(event)">All</li>
+            <li data-region="africa" onclick="filterByRegion(event)">Africa</li>
+            <li data-region="americas" onclick="filterByRegion(event)">America</li>
+            <li data-region="asia" onclick="filterByRegion(event)"v>Asia</li>
+            <li data-region="europe" onclick="filterByRegion(event)">Europe</li>
+            <li data-region="oceania" onclick="filterByRegion(event)">Oceania</li>
```

Comment:
consider to insert the `li` elements from the js code with `map()`. 
Imagine you have a large list of regions - we will want the computer to calculate the presented list and not to insert it one by one,